MiniSpotify/MiniSpotify/Source/APIRequestor.cs

```diff
@@ -52,6 +52,9 @@ public static APIRequestor Instance
         private string m_clientID = "93f2598a9eaf4056b34f7b5ca254ff17";
         private string m_clientSecret = "";
         private string m_clientSecretPath = "\\Assets\\Files\\ClientSecret.txt";
+        
+        private string t_albumimage = "";
+        public static string t_id = "";
```

Comment:
```suggestion
        public static string m_id = "";
```
Same with this one, let's keep the standards up.

Also, what's the reasoning behind using a static variable here?

Reply:
We don't use this variable anymore, please remove this.

Reply:
Fixed it with the last commit.